allennlp/tests/data/dataset_readers/text_classification_json_test.py

```diff
@@ -77,11 +77,12 @@ def test_read_from_file_ag_news_corpus_and_segments_sentences_properly(self, laz
         instances = reader.read(ag_path)
         instances = ensure_list(instances)
 
-        instance1 = {"tokens": [['Memphis', 'Rout'],
-                                ['Still', 'Stings', 'for', 'No', '.', '14',
-                                 'Louisville', ';', 'Coach', 'Petrino', 'Vows', 'to', 'Have',
+        instance1 = {"tokens": [['Memphis', 'Rout', 'Still', 'Stings', 'for', 'No', '.', '14',
```

Comment:
@kernelmachine, is this an improvement or a regression?

Reply:
@kernelmachine, unless I hear concerns from you about this, I'm going to merge this later today.

Reply:
this looks good to me.